android/settings.gradle

```diff
@@ -1 +1,6 @@
-rootProject.name = 'purchases-hybrid-common'
\ No newline at end of file
+rootProject.name = 'purchases-hybrid-common'
+sourceControl {
+    gitRepository("https://github.com/revenuecat/purchases-android.git") {
+        producesModule("com.revenuecat.purchases:purchases")
+    }
```

Comment:
found this and it seems to work as expected. well, other than the fact that it doesn't compile XD but it seems unrelated

Reply:
wutttttt this is great

Reply:
ok, so I think this is actually the reason why it's not working. It's related to that issue I mentioned where all the classes are being accessible when downloading the SDK from Maven. 

`PlatformInfo` and `Iso8601Utils` are not in the `:public` not in `:purchases` module. If you look at the build.gradle in `:purchases`:

```
dependencies {
    implementation fileTree(include: ['*.jar'], dir: 'libs')
    implementation project(":common")
    implementation project(":utils")
    implementation project(":feature:subscriber-attributes")
    implementation project(":feature:google")
    implementation project(":feature:identity")
    implementation project(":strings")
    api project(":public")
```

The only one we expose when someone adds the module `:purchases` is `:public`, so when building the project locally (and it looks like when downloading from git too) `purchases-hybrid-common` shouldn't have access to classes outside `:purchases` or `:public`. When adding `purchases` from maven, everything is exposed.

This is actually how I found out that everything was being exposed. I tried to add the local dependency and `PlatformInfo` was not accessible. But when downloading from Maven, it is.

Reply:
bummer, but that makes total sense. It also does mean that we don't have a great way of testing this while developing other than issuing a pre-release, right? 
let's sync up about it today